plasmapy/formulary/collisions.py

```diff
@@ -2033,3 +2039,91 @@ def coupling_parameter(
         )
 
     return coulomb_energy / kinetic_energy
+
+
+class CollisionFrequencies:
+    @validate_quantities(
+        v_a={"none_shall_pass": True},
+        T_a={
+            "none_shall_pass": True,
+            "can_be_negative": False,
+            "equivalencies": u.temperature_energy(),
+        },
+        n_b={"can_be_negative": False},
+        T_b={"can_be_negative": False, "equivalencies": u.temperature_energy()},
+    )
+    @particles.particle_input
+    def __init__(
+        self,
+        test_particle: particles.Particle,
+        field_particle: particles.Particle,
```

Comment:
Are `test_particle` and `field_particle` better left as individual parameters or should I do this as a tuple??

Reply:
Actually, I could probably use the GenericPlasma class

Reply:
Hm, not sure if GenericPlasma is a good fit here, since both test and field particles could be ions or electrons? 

I like having them as separate parameters to make it really explicit which is which (since that's very important)

Reply:
Yeah there seem to have been some efforts for some sort of species class awhile ago that would extend functionality of the particle class to include number density and temperature, though I don't think it was ever implemented.

That would probably be our best bet. But yeah, I agree, probably best to keep them separate